common/src/main/java/com/lightstep/tracer/shared/Util.java

```diff
@@ -53,4 +54,12 @@ static long fromHexString(String hexString) {
     static String toHexString(long l) {
         return Long.toHexString(l);
     }
+
+    static boolean IsNotMetaSpan(Span span) {
```

Comment:
Out of curiosity: Any reason to not put this directly in `Span`?

Reply:
I've been putting this in utils in other packages since it's not really part of a canonical span.